apps/labs/posts/sangho-internship-blogpost.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Sangho's Internship at Quansight"
+published: October 6, 2022
+author: sangho-lee
+description: 'Blogpost of working on the PyTorch-Ignite project during internship at Quansight'
+category: [Machine Learning, OSS Experience]
+featuredImage:
+  src: /posts/sangho-internship-blogpost/sangho1.png
+  alt: 'An image of the assigned mentors.'
+hero:
+  imageSrc: /posts/sangho-internship-blogpost/ignite_logo_mixed.svg
```

Comment:
The logo is being cropped at the top. Maybe we can upload a picture with more white space so that doesn't happen?

Reply:
I'm not sure because I do not find other images. Let me try with `png` format and if it happens again, please let me know :)